test/bigtest/network/scenarios/fetch-match-profiles-success.js

```diff
@@ -0,0 +1,232 @@
+
+export default server => {
+  server.create('match-profile', {

```

Comment:
Like we discussed if there is a way to reduce the number of records to create for testing please remove them.